### ISSUE TITLE
Impossible to remove "tcp-udp Generic" connection

### ISSUE THREAD

**Issue by kestouf** [bitfocus/companion#2291]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I have an "tcp-udp Generic" connection.
When I clic on "DELETE" nothing to passe !

In Log:
2023-01-23T09:29:54.600Z,"Internal","error","Error   in client handler 'instance_delete': TypeError: Cannot read property '1' of   undefined"

### Steps To Reproduce

1- clic on DELETE

### Environment (please complete the following information)

```markdown
OS: Windows 10
Browser: Chrome 109
Companion Version: 2.4.1 (2.4.1+4898-v2-4.1-9646192a)
```

**Comment by Julusian** [comment]
Im not able to reproduce this, and cant spot where this could be from.

Any chance of an export of your config?

**Comment by Muzakashi** [comment]
I'm having a similar issue, but with the yamaha-RCP module.
Getting the following message in the log when trying to delete the connection:
`Internal: Error in client handler 'instance_delete': TypeError: Cannot read property '10' of undefined`

**Comment by Julusian** [comment]
3.0.0-rc1 has been released, so we will not be doing another 2.4 release. This issue is fixed in 3.0
